websites/M/MangaFire/metadata.json

```
@@ -0,0 +1,38 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "135058085386387457",
+		"name": "Extiriority"
+	},
+	"service": "MangaFire",
+	"description": {
+		"en": "MangaFire.to is a completely free online modern manga platform with zero pop-up ads where we offer an extensive collection of Manga, Light-Novel, Manhwa and many more to read.",
+		"nl": "MangaFire.to is een compleet gratis online moderne manga platform zonder pop-up advertentie waar wij een groot collectie aanbieden zoals Manga, Light-Novel, Manhwa en veel meer om te lezen."
+	},
+	"url": "mangafire.to",
```

Website uses `www` subdomain.
```suggestion
	"url": [
		"mangafire.to",
		"www.mangafire.to"
	],
```